set.mm

```diff
@@ -17804,32 +17805,32 @@ disjoint variable condition ( ~ sb2 ).  Theorem ~ sb6f replaces the
   $}
 
   $( Extend wff definition to include atomic formulas with the membership
-     predicate.  This is read " ` x ` is an element of ` y ` ", " ` x ` is a
-     member of ` y ` ", " ` x ` belongs to ` y ` ", or " ` y ` contains
-     ` x ` ".  Note:  The phrase " ` y ` includes ` x ` " means " ` x ` is a
-     subset of ` y ` "; to use it also for ` x e. y ` , as some authors
-     occasionally do, is poor form and causes confusion, according to George
-     Boolos (1992 lecture at MIT).
+     predicate.  This is read either " ` x ` is an element of ` y ` ",
+     or " ` x ` is a member of ` y ` ", or " ` x ` belongs to ` y ` ",
+     or " ` y ` contains ` x ` ".  Note:  The phrase " ` y ` includes
+     ` x ` " means " ` x ` is a subset of ` y ` "; to use it also for
+     ` x e. y ` , as some authors occasionally do, is poor form and causes
+     confusion, according to George Boolos (1992 lecture at MIT).
 
      This syntactic construction introduces a binary non-logical predicate
-     symbol ` e. ` (stylized lowercase epsilon) into our predicate calculus.
-     We will eventually use it for the membership predicate of set theory, but
-     that is irrelevant at this point: the predicate calculus axioms for ` e. `
-     apply to any arbitrary binary predicate symbol.  "Non-logical" means that
-     the predicate is presumed to have additional properties beyond the realm
-     of predicate calculus, although these additional properties are not
-     specified by predicate calculus itself but rather by the axioms of a
-     theory (in our case set theory) added to predicate calculus.  "Binary"
-     means that the predicate has two arguments.
-
-     (Instead of introducing ~ wel as an axiomatic statement, as was done in an
+     symbol ` e. ` into our predicate calculus.  We will eventually use it for
```

Comment:
Why did you remove "(stylized lowercase epsilon)"?

Reply:
It's in the comment of the $c-statement introducing it.  For most other symbols, there is no description of what they look like (or only in a comment of the $c-statement).  It is displayed in the HTML page, for people to see what it looks like.

That said, I can put it back, though the addition looks a bit out of place to me.

Reply:
Since the comment for the  $c-statement is not dsiplayed in the HTML page, I would prefer to have this put back.

Reply:
Change reverted.